salt/states/cmd.py

```diff
@@ -171,7 +171,7 @@
 
 .. note::
 
-    Use ``cmd.run`` together with :mod:`onchanges </ref/states/requisites#onchanges>`
+    Use :mod:`cmd.run <salt.states.cmd.run>` together with :mod:`onchanges </ref/states/requisites#onchanges>`
     instead of ``cmd.wait``.
```

Comment:
Can we also make this into a link to match the link for cmd.run? This can be changed to:

```
:mod:`cmd.wait <salt.states.cmd.wait>`
```